src/Http/Controllers/ResetController.php

```diff
@@ -2,21 +2,35 @@
 
 namespace Whitecube\LaravelCookieConsent\Http\Controllers;
 
-use Whitecube\LaravelCookieConsent\CookiesManager;
+use Illuminate\Support\Facades\Cookie;
 use Illuminate\Http\Request;
+use Whitecube\LaravelCookieConsent\CookiesManager;
+use Whitecube\LaravelCookieConsent\Facades\Cookies;
+
 
-class ResetController
-{
-    public function __invoke(Request $request, CookiesManager $cookies)
-    {
-        $response = ! $request->expectsJson()
+class ResetController {
+
+    public function __invoke(Request $request, CookiesManager $cookies) {
+        $response = !$request->expectsJson()
```

Comment:
Please do not use your own coding style rules within the package (I guess you're using an automatic CS fixer with different rules than ours). Thanks !